x-pack/platform/packages/shared/onechat/onechat-common/agents/definition.ts

```diff
@@ -75,6 +75,13 @@ export interface AgentConfiguration {
    * For custom per-step instructions, use the `research` and `answer` configuration fields instead.
    */
   instructions?: string;
+  /**
+   * If set to true, the custom instructions will be used as a replacement for the system prompt instead of extending it.
+   *
+   * This will impact bah the research and answer prompts. For custom per-step instructions, use the `research` and `answer` configuration fields instead.
```

Comment:
```suggestion
   * This will impact both the research and answer prompts. For custom per-step instructions, use the `research` and `answer` configuration fields instead.
```

Reply:
😄 thank you